### PR TITLE
KAFKA-6515 Adding toString() method to o.a.k.connect.data.Field

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://issues.apache.org/jira/browse/KAFKA-6515

### Committer Checklist (excluded from commit message)
- [x] Verify design and implementation 
- [x] Verify test coverage and CI build status
- [x] Verify documentation (including upgrade notes)

### CODE REVIEW

File: connect/api/src/main/java/org/apache/kafka/connect/data/Field.java [hunk 1]
```diff
@@ -73,4 +73,13 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(name, index, schema);
     }
+
+    @Override
+    public String toString() {
+        return "Schema{" +
```
Comment (bartektartanus) [inline connect/api/src/main/java/org/apache/kafka/connect/data/Field.java]: Schema? It's Field class...

Comment (gunnarmorling) [reply connect/api/src/main/java/org/apache/kafka/connect/data/Field.java]: Of course. Updated it. Thanks!

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-test-coverage/267/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-test-coverage/268/

**Comment by gunnarmorling** [comment]
Hi @rhauch, @ewencp, feedback/merge welcome on that tiny improvement whose issue I spotted in JIRA and which also had been bugging me. Thanks!

**Review by hachikuji — APPROVED** [review]
Thanks for the patch. LGTM

**Pull request merged**